renderdoc/driver/vulkan/vk_core.cpp

```diff
@@ -1277,13 +1277,26 @@ bool WrappedVulkan::EndFrameCapture(void *dev, void *wnd)
       uint32_t stride = fmt.compByteWidth * fmt.compCount;
 
       bool buf1010102 = false;
+      bool buf565 = false, buf5551 = false;
       bool bufBGRA = (fmt.bgraOrder != false);
 
-      if(fmt.special && fmt.specialFormat == eSpecial_R10G10B10A2)
-      {
-        stride = 4;
-        buf1010102 = true;
-      }
+      if(fmt.special)
```

Comment:
I'd like braces on this `if` please, around a long scope like the `switch` statement.